elasticsearch/templates/activity_definition.json

```diff
@@ -41,6 +41,26 @@
             "dateFields": {
               "type": "keyword"
             },
+            "groups": {
```

Comment:
So when we do a release, we'll need to update the template in ES and re-upload all the activity definitions?

Reply:
yes yufeng